react-frontend/src/components/Nav/breadcrumbs.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useLocation, useHistory } from 'react-router-dom';
+import '../../css/breadcrumbs.css';
+
+const routeDisplayNames = {
+  resources: 'Resources',
+  'products-services': 'Products and Services',
+  'digital-framework': 'Digital Framework',
+  'case-studies': 'Case Studies',
+};
+const BreadCrumbs = () => {
+  const history = useHistory();
+  const routeLocation = useLocation();
+  const [pathName, setPathName] = useState(routeLocation.pathname);
+
+  useEffect(() => {
+    return history.listen((location) => {
```

Comment:
can you explain whats going on here? It appears as this component unmounts this function is called

Reply:
I'll change this so it doesn't trigger on unmount!

Reply:
can you explain whats going on here?

Reply:
still looks like its on unmount